iTechArt.CinemaWebApp.API/Models/Policies.cs

```diff
@@ -0,0 +1,20 @@
+﻿using Microsoft.AspNetCore.Authorization;
+
+namespace iTechArt.CinemaWebApp.API.Models
+{
+    public class Policies
+    {
+        public const string Admin = "Admin";
+        public const string User = "User";
+
+        public static AuthorizationPolicy AdminPolicy()
+        {
+            return new AuthorizationPolicyBuilder().RequireAuthenticatedUser().RequireRole(Admin).Build();
```

Comment:
Please place chaining method calls from the new line
```suggestion
            return new AuthorizationPolicyBuilder()
                .RequireAuthenticatedUser()
                .RequireRole(Admin)
                .Build();
```